### PR TITLE
Add Cake Aliases and Publishing package to local registry

### PR DESCRIPTION
**Pull request by ChrisTTian667**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,100 @@
 # cake-git-ext-plugin
-Additional Git functionality like change detection for Cake scripts
+
+[![GitHub license](https://img.shields.io/github/license/ChrisTTian667/cake-git-ext-plugin.svg)](https://github.com/ChrisTTian667/cake-git-ext-plugin/main/LICENSE)
```
Comment (hoffmann-joern) [inline README.md]: Add a LICENSE file to this PR and link to it.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,100 @@
...
+
+```csharp
+ICakeContext context;
+var hasMatch = context.GitGetChangeMatchPattern(changes, "your-regex-pattern");
+```
+
+## Usage
+Add the CakeExt.Git NuGet package to your Cake build script project.
+```csharp
+#addin "nuget:?package=CakeExt.Git"
```
Comment (hoffmann-joern) [inline README.md:94]: Should we add a note that we publish to the GitHub Nuget feed and not to "nuget.org"?

File: build.cake [hunk 3]
```diff
@@ -0,0 +1,131 @@
+#tool "dotnet:?package=GitVersion.Tool&version=5.12.0"
+#tool dotnet:?package=CycloneDX&version=2.8.1
```
Comment (hoffmann-joern) [inline build.cake]: "CycloneDX" and "Docker" are not used.

File: README.md (hunk 1 above)
Comment (ChrisTTian667) [reply README.md]: There already is a Licence file linked below. But fixed the Link :)

File: build.cake (hunk 3 above)
Comment (ChrisTTian667) [reply build.cake]: Removed

**Review by hoffmann-joern — APPROVED** [review]

**Pull request merged**